### ISSUE TITLE
Vue SPA - 'AuthSdkError: Unable to parse a token from the url' after upgrade to v2.0.0

### ISSUE THREAD

**Issue by maneeshsahu** [okta/okta-oidc-js#765]
## I'm submitting this issue for the package(s):

- [ ] jwt-verifier
- [ ] okta-angular
- [ ] oidc-middleware
- [ ] okta-react
- [ ] okta-react-native
- [x] okta-vue

## I'm submitting a:

- [x] Bug report  
- [ ] Feature request
- [ ] Other (Describe below)

## Current behavior
After I did a npm check updates (ncu -u), my package.json updated with 
`"@okta/okta-vue": "^2.0.0",`

After that my OIDC callbacks after a successful auth resulted in this error in the console:
`AuthSdkError: Unable to parse a token from the url `

When I rolled back to 1.3.0, it worked as before.

## Expected behavior
OIDC callback should work as before. Successful auth should result in the tokens being consumed by the Vue App.

## Minimal reproduction of the problem with instructions
Vue OIDC Sample with sign-in widget.

## Extra information about the use case/user story you are trying to implement
Building a Vue SPA

## Environment

- Package Version: 2.0.0
- Browser: Latest Chrome, Firefox and Safari
- OS: MacOS Catalina
- Node version (`node -v`): v13.12.0
- Other:
Used in concert with okta-signin-widget 
`"@okta/okta-signin-widget": "^3.9.1",`

**Comment by maneeshsahu** [comment]
Github Repo: https://github.com/oktadeveloper/byob-dashboard/blob/master/package.json#L12

**Comment by swiftone** [comment]
@maneeshsahu - Thanks for the report.  Do you have the same problem using our newly released widget version 4.0?

**Comment by maneeshsahu** [comment]
After upgrading the dependencies in package.json, I have the scenario working again. 

```
 @okta/okta-signin-widget   ^3.9.2  →   ^4.0.1 
 @okta/okta-vue             ^1.3.0  →   ^2.0.0 
```
